RiotSharp/StaticRiotApi.cs

```diff
@@ -789,11 +813,11 @@ public RealmStatic GetRealm(Region region)
             return wrapper.SummonerSpellListStatic;
         }
     
-        public SummonerSpellStatic GetSummonerSpell(Region region, SummonerSpell summonerSpell,
+        public SummonerSpellStatic GetSummonerSpell(Region region, int summonerSpellId,
             SummonerSpellData summonerSpellData = SummonerSpellData.basic, Language language = Language.en_US)
         {
             var wrapper = cache.Get<string, SummonerSpellStaticWrapper>(
-                SummonerSpellCacheKey + summonerSpell.ToCustomString());
+                SummonerSpellCacheKey + summonerSpellId.ToString());
```

Comment:
if you only keep the spell id as int, you don't need `ToString` when doing concatenation